10.9-libcxx/stable/main/finkinfo/sci/netcdf-cxx4.info

```diff
@@ -28,7 +28,8 @@ Replaces: <<
 <<
 
 Source: https://github.com/Unidata/%n/archive/v%v.tar.gz
-Source-MD5: 3bc361ef3196163ae1006c32fdf5b35c
+Source-MD5: b4a9783b2b0b98d4e6f36cc19c8d08ef
+Source-Checksum: SHA1(796d71c7aae8bc15e923d5c5b50a54bcc7538f0c)
```

Comment:
Checksum mismatch for netcdf-cxx-4.3.0? http://bindist.finkproject.org/10.13/logs/netcdf-cxx4_4.3.0-1_300a93d9e4a2348414a3ebe3fc4f1c0e.log
````
Expected: 796d71c7aae8bc15e923d5c5b50a54bcc7538f0c
Actual: MD5(f840a75d3c68390fb7392be5c3f663cd)
        SHA1(c3290c514cfb414b5316a03e9cee1a48440d373c)
        SHA256(25da1c97d7a01bc4cee34121c32909872edd38404589c0427fefa1301743f18f)
````